src/app/models/propTypes/metadata/index.js

```diff
@@ -27,13 +27,13 @@ export const mediaMetadataPropTypes = {
 };
 
 export const optimoMetadataPropTypes = {
-  id: string.isRequired,
+  id: string,
```

Comment:
Kinda feel like this should be required as it is needed to render the `ArticleMetadata` component. Is there an error thats being emitted if this is kept as `.isRequired`?